server/server.go

```diff
@@ -510,17 +510,26 @@ func (s *Server) startMonitoring(secure bool) error {
 		hp           string
 		err          error
 		httpListener net.Listener
+		port   int
```

Comment:
I see server start panics quite a bit in unit tests. Intuition tells me this is a lingering port, especially if tests are reusing ports. I'm not sure what else would cause those panics?